src/backend/distributed/transaction/backend_data.c

```diff
@@ -444,6 +461,35 @@ StoreAllActiveTransactions(Tuplestorestate *tupleStore, TupleDesc tupleDescripto
 }
 
 
+/*
+ * CurrentUserHasPermissionToViewStatsOf returns true if currentUserId can
+ * see backends of backendOwnedId.
+ *
+ * We follow the same approach with pg_stat_activity.
+ */
+static
+bool
+CurrentUserHasPermissionToViewStatsOf(Oid currentUserId, Oid backendOwnedId)
```

Comment:
Maybe CurrentUserHasPermissionToViewStatsOf -> UserHasPermissionToViewStatsOf ? (Since we're passing the current user's ID)